packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.h

```diff
@@ -246,63 +246,75 @@ namespace Ioss {
       flush_database_nl();
     }
 
-    /** \brief If a database type supports groups and if the database
-     *         contains groups, open the specified group.
+    /** \brief If a database type supports internal change sets and if the database
+     *         contains internal change sets, open the specified set.
      *
-     *  If the group_name begins with '/', it specifies the absolute path
-     *  name from the root with '/' separating groups.  Otherwise, the
-     *  group_name specifies a child group of the currently active group.
-     *  If group_name == "/" then the root group is opened.
-     *
-     *  \param[in] group_name The name of the group to open.
+     *  \param[in] set_name The name of the set to open.
      *  \returns True if successful.
      */
-    bool open_group(const std::string &group_name)
+    bool open_internal_change_set(const std::string &set_name)
     {
       IOSS_FUNC_ENTER(m_);
-      return open_group_nl(group_name);
+      return open_internal_change_set_nl(set_name);
```

Comment:
Not sure every function with a lock (`IOSS_FUNC_ENTER(m_)`) needs a non-lock _nl version.  The _nl version is only needed if you need to call the function from another function which already holds a lock...